ui/docker/40-envsubst-on-reearth_config.sh

```
@@ -0,0 +1,42 @@
+set -e
+
+ME=$(basename "$0")
+
+entrypoint_log() {
+    if [ -z "${NGINX_ENTRYPOINT_QUIET_LOGS:-}" ]; then
+        echo "$@"
+    fi
+}
+
+auto_envsubst() {
+    local template_file="/tmp/reearth_config.json.tmpl"
```

Shellcheck

[warning] In POSIX sh, 'local' is undefined.

(SC3043)